auth-web/src/assets/styles/theme.scss

```diff
@@ -0,0 +1,69 @@
+// Gray Palette
```

Comment:
A full, duplicate copy of the theme variables isn't what I had in mind.

I thought this file would only contain:
```
import "@/assets/scss/theme.scss"
```

Reply:
ah i see you what you mean now.

Reply:
Done in next commit